docs/rfcs/0623-redis-service.md

```diff
@@ -48,20 +48,35 @@ Redis offers a key-value view, so the path of files could be represented as the
 The content of file will be represented directly as `String`, and metadata will be encoded as [`bincode`](https://github.com/bincode-org/bincode.git) before storing as `String`.
 
 ```
-Object: /home/monika/poem0.txt
-
-content:  Key: v0:c:/home/monika/poem0.txt ─┐
-                                            │
-metadata: Key: v0:m:/home/monika/poem0.txt  │
-│                                           ▼
-│   STRING                                STRING
-└► ┌──────────────────────┐              ┌────────────────────┐
-   │\x00\x00\x00\x00\xe6\a│              │1JU5T3MON1K413097321│
-   │\x00\x00\xf8\x00\a4)!V│              │&JU5$T!M0N1K4$%#@#$%│
-   │\x81&\x00\x00\x00Q\x00│              │3231J)U_ST#MONIKA@#$│
-   │         ...          │              │1557(m0N1ka3just4M  │
-   └──────────────────────┘              │      ...           │
-                                         └────────────────────┘```
++------------------------------------------+
+|Object: /home/monika/                     |
+|                                          |           SET
+|child: Key: v0:k:/home/monika/           -+---------> 1) /home/monika/poem0.txt
+|                                          |
+|/* directory has no content  */           |
+|                                          |
+|metadata: Key: v0:m:/home/monika/         |
++------------------------------------------+
+
++------------------------------------------+
+|Object: /home/monika/poem0.txt            |
+|                                          |           SET
+|child: Key: v0:k:/home/monika/poem0.txt  -+---------> <empty>
```

Comment:
Maybe we don't need to set `child` for the `file` object?

Reply:
Yes.